avocado/utils/multipath.py

```diff
@@ -85,7 +85,8 @@ def get_mpath_name(wwid):
     """
     if device_exists(wwid):
         cmd = "multipath -l %s" % wwid
-        return process.system_output(cmd, sudo=True).split()[0]
+        return process.run(cmd,
+                           sudo=True).stdout_text.split()[0]
```

Comment:
Hi @narasimhan-v, Just a small suggestion. Not sure if @clebergnu agree with that, but:

Looks like that `process.run().stdout_text` is being used so many times here that we could have a small private method here or inside process module, just to get a better organization.

Reply:
The `process.run().stdout_text` pattern is used all over Avocado, so let's defer that to a different discussion.  @beraldoleal if you have any ideas, please go ahead an send a separate proposal. Thanks!